packages/babel-plugin-proposal-unicode-sets-regex/test/fixtures/transform-u/string-properties/output.js

```diff
@@ -0,0 +1 @@
+/(?:\*️⃣|#️⃣|0️⃣|1️⃣|2️⃣|3️⃣|4️⃣|5️⃣|6️⃣|7️⃣|8️⃣|9️⃣)/;
```

Comment:
Emojis in code files, I like it :)

Reply:
I chose `\p{Emoji_Keycap_Sequence}` here because 1) the result is short, and 2) it is likely more stable across Unicode versions than other string properties, which involves much more emojis.